model/_plugins/notifications/delete_config/structures.smithy

```diff
@@ -0,0 +1,33 @@
+//  SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@xDataType("array")
+@pattern("^(?!_|template|query|field|point|clear|usage|stats|hot|reload|painless).+$")
+@documentation("Comma-separated list of notifications channels ids.")
+string PathNotificationsChannelsIds
+
+@input
+structure NotificationsConfigs_Delete_WithParams_Input {
+    @httpQuery("config_id")
+    config_id: String,
+
+    @httpQuery("config_id_list")
+    config_id_list: PathNotificationsChannelsIds
+}
+
+@input
+structure NotificationsConfigs_Delete_Input {
+    @required
+    @httpLabel
+    config_id: String
+}
+
+// TODO: Fill in Output Structure (delete_response_list: object)
+structure NotificationsConfigs_Delete_Output {
+}
```

Comment:
When you fill this out later, check out Smithy [Map](https://smithy.io/2.0/spec/aggregate-types.html?highlight=union#map)